config/react-client.tf

```diff
@@ -15,13 +15,13 @@ resource "keycloak_openid_client" "react_client" {
   direct_access_grants_enabled             = true
   service_accounts_enabled                 = false
 
-  admin_url = "https://web.dev.callisto.homeoffice.gov.uk/"
-  root_url  = "https://web.dev.callisto.homeoffice.gov.uk/"
```

Comment:
It appears to me that we’re not actually using these:
https://registry.terraform.io/providers/mrparkers/keycloak/latest/docs/resources/openid_client#admin_url
https://registry.terraform.io/providers/mrparkers/keycloak/latest/docs/resources/openid_client#root_url